src/SyncTrayzor/Pages/AboutView.xaml

```diff
@@ -6,6 +6,7 @@
         Height="400" Width="250"
         WindowStyle="None" SizeToContent="Height"
         ShowInTaskbar="False"
+        ResizeMode="CanMinimize"
```

Comment:
I think CanMinimize is misleading here - it implies that the window can be minimized when, due to the other properties on it, it can't be.